### PR TITLE
Feature/vpc securitygroup cicd setup

### PR DESCRIPTION
**Pull request by LH-Song**

### CODE REVIEW

File: infrastructure/environments/dev/main.tf
Comment (markma85) [inline infrastructure/environments/dev/main.tf:1]: Suggest using only public subnet for dev env.

File: .github/workflows/terraform.yml
Comment (markma85) [inline .github/workflows/terraform.yml:1]: What are the differences between `main` and `production` branches? Suggest keeping only `main`.

Comment (LH-Song) [reply .github/workflows/terraform.yml:1]: You right

**Review by markma85 — APPROVED** [review]
Looks good to me! The suggested comments are optional and not mandatory for changes—feel free to implement them as you see fit.

File: infrastructure/environments/dev/main.tf
Comment (LH-Song) [reply infrastructure/environments/dev/main.tf:1]: Yes, but for 
1.Early Detection of Issues
2.Improved Security Practices
I prefer to keep both.

**Pull request merged**